### PR TITLE
Minor APIView improvements for Java

### PR DESCRIPTION
**Pull request by JonathanGiles**
Three fixes to the Java APIView processor:

1) Remove unnecessary newline on the maven dependencies list
2) Remove all anonymous inner class code from appearing in public static field declarations, as it was just adding a lot of noise.
3) Remove duplicate "missing JavaDoc" error messages

### CODE REVIEW

**Comment by praveenkuttappan** [comment]
Can you please update the parser version if this change needs to update existing reviews? Please also update the parser version in JavaLanguageService.cs and APIView.csproj file.

**Review by praveenkuttappan — APPROVED** [review]

File: src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/analysers/JavaASTAnalyser.java [hunk 1]
```diff
@@ -798,8 +798,13 @@ private void tokeniseFields(boolean isInterfaceDeclaration, TypeDeclaration<?> t
...
                     if (variableDeclaratorOption.isPresent()) {
-                        addToken(SPACE, new Token(PUNCTUATION, "="), SPACE);
-                        addToken(new Token(TEXT, variableDeclaratorOption.get().toString()));
+                        Expression e = variableDeclaratorOption.get();
+                        if (e.isObjectCreationExpr() && e.asObjectCreationExpr().getAnonymousClassBody().isPresent()) {
+                            // no-op because we don't want to include all of the anonymous inner class details
+                        } else {
+                            addToken(SPACE, new Token(PUNCTUATION, "="), SPACE);
+                            addToken(new Token(TEXT, variableDeclaratorOption.get().toString()));
+                        }
```
Comment (srnagar) [inline src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/analysers/JavaASTAnalyser.java:807]: Can we inverse the `if` condition to remove the need for `else`?

**Review by srnagar — APPROVED** [review]

**Comment by JonathanGiles** [comment]
> Can you please update the parser version if this change needs to update existing reviews? Please also update the parser version in JavaLanguageService.cs and APIView.csproj file.

@praveenkuttappan Will do - although I'm curious if we could make this simpler? I always wish there was just one file to update, and it's now even worse because I have to update the XmlLanguageService too! :-) And I always need to hunt around for the right files (e.g. it isn't APIView.csproj, it's APIViewWeb.csproj, but I only realised that after I opened the wrong one) :-)

**Comment by JonathanGiles** [comment]
@praveenkuttappan Please let me know if the version string updates are correct (I'm never entirely sure when I should update the json-output versionString value)

**Comment by JonathanGiles** [comment]
Ping @praveenkuttappan - thanks!

**Comment by JonathanGiles** [comment]
This is failing for reasons unrelated to the PR - can you get it merged and deployed sometime please @praveenkuttappan?

**Pull request merged**